### PR TITLE
docs: add python courses and books in indonesia language

### PR DESCRIPTION
**Pull request by okzapradhana**
## What does this PR do?
Add resources (courses and books) for Python in Indonesia language.

### Why is this valuable (or not)?
It is recommended by the community and me personally.

### How do we know it's really free?
There is no any prerequisite to access the course, they provide to us for free.

### For book lists, is it a book? For course lists, is it a course? etc.
Book and course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -158,10 +158,13 @@
 ### Python
 
 * [Belajar Python](http://www.belajarpython.com)
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia)
```
Comment (davorpa) [inline books/free-programming-books-id.md:161]:
```suggestion
* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
```

File: books/free-programming-books-id.md [hunk 2]
```diff
@@ -158,10 +158,13 @@
...
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
 * [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan *(Membutuhkan akun Google Play Books atau email yang valid)*
 * [Database PostgreSQL, Pemrograman Python, dan SMS Gateway](http://rab.co.id/files/python/bukupython2.pdf.gz) (PDF)
 * [Kursus Singkat Machine Learning dengan TensorFlow API](https://developers.google.com/machine-learning/crash-course?hl=id)
+* [Python](https://codesaya.com/python/)
```
Comment (davorpa) [inline books/free-programming-books-id.md:166]:
```suggestion
* [Python](https://codesaya.com/python/) - CodeSaya
```

File: books/free-programming-books-id.md [hunk 3]
```diff
@@ -158,10 +158,13 @@
...
+* [Tutorial Dasar Pemrograman Python](https://www.petanikode.com/tutorial/python/)
```
Comment (davorpa) [inline books/free-programming-books-id.md:167]:
```suggestion
* [Tutorial Dasar Pemrograman Python](https://www.petanikode.com/tutorial/python/) - Petani Kode, Ahmad Muhardian
```

File: courses/free-courses-id.md [hunk 4]
```diff
@@ -327,12 +328,15 @@
 
 ### Python
 
+* [#LiveSeminggu (Python Dasar)](https://www.youtube.com/playlist?list=PLl-Zj2iuqlwviXEWkyT08NbK54kpa5cvu) - NgodingPython
 * [Belajar Pemrograman Python untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH_HY6bL3JER8WJOxz1nb3_H) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
 * [Belajar Python Bahasa Indonesia [Dasar]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo7cgStrKAMhgnOT66z2qKz1) - Kelas Terbuka
 * [Belajar Python Bahasa Indonesia [Versi Baru]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo59lUBwU-XHH7Ymmb04ffOY) - Kelas Terbuka
 * [Intro to Python](https://www.youtube.com/playlist?list=PLM4Qh7PEStOmBHR6Ey0BzPf3XNsPuSXL-) - Tensaitech Academy
 * [OpenCV untuk Pemula](https://www.youtube.com/playlist?list=PLl-Zj2iuqlwt4cBsBy_Ej7gNlXdQ7blCB) - NgodingPython
+* [Python (dasar)](https://www.youtube.com/playlist?list=PLT2AwM1dOOUe27iCPe68Z0qwwDeNyJawN) - beecoder id *(Dalam Proses)*
```
Comment (davorpa) [inline courses/free-courses-id.md:337]:
```suggestion
* [Python (dasar)](https://www.youtube.com/playlist?list=PLT2AwM1dOOUe27iCPe68Z0qwwDeNyJawN) - beecoder id *(:construction: Dalam Proses)*
```

File: courses/free-courses-id.md [hunk 5]
```diff
@@ -327,12 +328,15 @@
...
 * [Belajar Pemrograman Python untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH_HY6bL3JER8WJOxz1nb3_H) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
 * [Belajar Python Bahasa Indonesia [Dasar]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo7cgStrKAMhgnOT66z2qKz1) - Kelas Terbuka
 * [Belajar Python Bahasa Indonesia [Versi Baru]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo59lUBwU-XHH7Ymmb04ffOY) - Kelas Terbuka
 * [Intro to Python](https://www.youtube.com/playlist?list=PLM4Qh7PEStOmBHR6Ey0BzPf3XNsPuSXL-) - Tensaitech Academy
 * [OpenCV untuk Pemula](https://www.youtube.com/playlist?list=PLl-Zj2iuqlwt4cBsBy_Ej7gNlXdQ7blCB) - NgodingPython
+* [Python (dasar)](https://www.youtube.com/playlist?list=PLT2AwM1dOOUe27iCPe68Z0qwwDeNyJawN) - beecoder id *(:construction: Dalam Proses)*
 * [Tutorial Belajar Python Pandas](https://www.youtube.com/playlist?list=PL2O3HdJI4voGdD_9xhVCTBoDTDNHpajm5) - Indonesia Belajar
+* [Tutorial Implementasi Rumus Matematika Di Python Untuk Pemula](https://www.youtube.com/watch?v=G18YEjpwHDY) - Dea Afrizal
```
Comment (davorpa) [inline courses/free-courses-id.md:339]: Too short video. Removing...

```suggestion
```

**Review by davorpa — APPROVED** [review]

File: books/free-programming-books-id.md [hunk 6]
```diff
@@ -158,10 +158,13 @@
 ### Python
 
 * [Belajar Python](http://www.belajarpython.com)
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
 * [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan *(Membutuhkan akun Google Play Books atau email yang valid)*
 * [Database PostgreSQL, Pemrograman Python, dan SMS Gateway](http://rab.co.id/files/python/bukupython2.pdf.gz) (PDF)
 * [Kursus Singkat Machine Learning dengan TensorFlow API](https://developers.google.com/machine-learning/crash-course?hl=id)
+* [Python](https://codesaya.com/python/) - CodeSaya
```
Comment (eshellman) [inline books/free-programming-books-id.md:166]: I would characterize this as a 1. interactive tutorial or 2. a course or 3. not really a book.

File: books/free-programming-books-id.md [hunk 7]
```diff
@@ -158,10 +158,13 @@
 ### Python
 
 * [Belajar Python](http://www.belajarpython.com)
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
```
Comment (eshellman) [inline books/free-programming-books-id.md:161]: this is mostly video content - doesn't it fit better in the courses list?

**Review by eshellman — CHANGES_REQUESTED** [review]
Please consider comments on which list they belong to.

File: books/free-programming-books-id.md (hunk 6 above)
Comment (okzapradhana) [reply books/free-programming-books-id.md:166]: got it, let me put in to course instead. thanks for the correction!

File: books/free-programming-books-id.md (hunk 7 above)
Comment (okzapradhana) [reply books/free-programming-books-id.md:161]: ah yes you're right, it's more suitable to put in course list. thankyou for the correction!

**Comment by okzapradhana** [comment]
Just updated according to requested changes, please let me know if there is anything that still missed. thankyou @eshellman

File: courses/free-courses-id.md [hunk 8]
```diff
@@ -354,12 +355,16 @@
...
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
 * [Belajar Python Bahasa Indonesia [Dasar]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo7cgStrKAMhgnOT66z2qKz1) - Kelas Terbuka
 * [Belajar Python Bahasa Indonesia [Versi Baru]](https://www.youtube.com/playlist?list=PLZS-MHyEIRo59lUBwU-XHH7Ymmb04ffOY) - Kelas Terbuka
 * [Intro to Python](https://www.youtube.com/playlist?list=PLM4Qh7PEStOmBHR6Ey0BzPf3XNsPuSXL-) - Tensaitech Academy
 * [OpenCV untuk Pemula](https://www.youtube.com/playlist?list=PLl-Zj2iuqlwt4cBsBy_Ej7gNlXdQ7blCB) - NgodingPython
+* [Python](https://codesaya.com/python/) - CodeSaya
+* [Python (dasar)](https://www.youtube.com/playlist?list=PLT2AwM1dOOUe27iCPe68Z0qwwDeNyJawN) - beecoder id *(Dalam Proses)*
 * [Tutorial Belajar Python Pandas](https://www.youtube.com/playlist?list=PL2O3HdJI4voGdD_9xhVCTBoDTDNHpajm5) - Indonesia Belajar
 * [Tutorial Python Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXDc9Wcsi2fUn0pAr41GDL6m) - Sekolah Koding
```
Comment (eshellman) [inline courses/free-courses-id.md:369]:
```suggestion
```
replaced from above

**Pull request merged**